src/main/kotlin/blackjack/domain/participantion/Players.kt

```diff
@@ -0,0 +1,15 @@
+package blackjack.domain.participantion
+
+import blackjack.domain.card.CardDeck
+import blackjack.domain.card.Cards
+
+class Players(val players: List<Player>) {
+
+    constructor(names: List<String>, cardDeck: CardDeck) : this(
+        names.map { name ->
+            val cards = Cards(cardDeck)
+
+            Player(name, cards)
+        }
+    )
```

Comment:
```suggestion
   constructor(names: List<String>, cardDeck: CardDeck) : this(
        names.map { name -> Player(name, Cards(cardDeck)) }
    )
```

한번에 표현해주면 읽기 더 쉽습니다 :)